doc/REL_NOTES

```diff
@@ -51,6 +51,10 @@ This Release
          - Add VMware for network ordering (BZ1024)
          - Fixes for byte order selecting.
          - Add ASCII and MacRoman as aliases for the ISO-8859-1 codeset
+         - Fix for bidirectional IIOP where the client creates a callback IOR with alternate
```

Comment:
There already is a bidir unit test. Note this condition requires a multi-homed host. In fact I diagnosed it using demo/bidir and debugging remotely with intellij.